docs/css/nav.css

```diff
@@ -50,13 +50,19 @@
   background-color: var(--dark-color);
   padding: 1rem 1rem;
   text-align: center;
-  color: white;
+  color: var(--white);
 }
 
 .footer .link {
-  color: white;
+  color: var(--white);
 }
 
 .footer ul {
   padding-inline-start: 0px;
 }
+@media (max-width: 767px) {
```

Comment:
Check media query referred above

Reply:
thanks for the suggestion, I'll make it 768px